android/app/src/main/java/com/sample/MainActivity.kt

```diff
@@ -22,11 +25,17 @@ class MainActivity : ReactActivity() {
     @Inject
     lateinit var manifestProvider: ManifestProvider
 
+    @Inject
+    lateinit var bundleNameProvider: ReactBundleNameProvider
+
+    private val testAppReactNativeHost: TestAppReactNativeHost
+        get() = reactNativeHost as TestAppReactNativeHost
+
     private val listener = { component: ComponentViewModel ->
         startActivity(
-                ComponentActivity.newIntent(
-                        this, component.name, component.displayName
-                )
+            ComponentActivity.newIntent(
+                this, component.name, component.displayName
```

Comment:
Can we settle on the default AS settings (whatever they are) to avoid unnecessary whitespace changes in the future?

Reply:
We should stick to the kotlin style guide, and I believe that the changes here are following that style. See my other comment in the closed PR.

Reply:
Can we check in the settings so we don't have to ask contributors to change their settings in AS?

Reply:
(I'm fine with a new PR for that.)

Reply:
Reverted the style changes.